ui/pages/onboarding-flow/import-srp/index.scss

```diff
@@ -1,37 +1,19 @@
 .import-srp {
-  width: 940px;
-
-  @media screen and (max-width: 349px) {
-    width: 300px;
-  }
-
-  @media screen and (min-width: 350px) and (max-width: 460px) {
-    width: 357px;
-  }
-
-  @media screen and (min-width: 461px) and (max-width: $break-small) {
-    width: 450px;
-  }
-
-  @media screen and (min-width: $break-small) and (max-width: 900px) {
-    width: 580px;
-  }
```

Comment:
We should try to avoid using a bunch of media queries that aren't in the system. 

## Screenshots

<img width="575" alt="Screen Shot 2022-04-20 at 3 36 35 PM" src="https://user-images.githubusercontent.com/8112138/164336643-b90d42ba-6ed5-4584-9dc2-4395ec3376af.png">
<img width="621" alt="Screen Shot 2022-04-20 at 3 41 08 PM" src="https://user-images.githubusercontent.com/8112138/164336645-0a028aa5-8fe1-49b6-8c32-7770bfe8e782.png">